build-scripts/ci_parser/ci_parser.py

```diff
@@ -19,11 +19,7 @@
 from junit_helpers import JUnitResultBuilder, JUnitTestCase, JUnitTestSuite, JUnitTestStatus, LOG_FILE_NAME
 from logging_helper import build_logger, mute_logging, CustomLogger
 
-try:
-    from bs4 import BeautifulSoup
-except ImportError:
-    print('bs4 not installed; make sure you have bs4 in your active python env.')
-    exit(1)
+from bs4 import BeautifulSoup
```

Comment:
🤷

Reply:
Rather than the full vomit of a stack trace on failed import in python, I was toying around with trying to make things a bit more friendly for the end user. Since this was being called by a shell script, that layer of indirection was leading to some unnecessary mental churn.

Reply:
will restore…